src/App.tsx

```diff
@@ -23,6 +24,7 @@ const Groups = React.lazy(() => import(/* webpackChunckName: "Groups" */ './page
 const ShoppingListsMainPage = React.lazy(
   () => import(/* webpackChunckName: "ShoppingListsMainPage" */ './pages/ShoppingListsMain'),
 )
+const Pair2MainPage = React.lazy(() => import(/* webpackChunckName: "ShoppingListsMainPage" */ './pages/Pair2'))
```

Comment:
update webpackchunckname

Reply:
fixed in commit 9dba039b3202249cd2782d2f05140593bbd22a95